lib/controllers/cats.js

```diff
@@ -1,15 +1,15 @@
 const { Router } = require('express');
 const Cat = require('../models/Cat');
 const router = Router();
-console.log('hello');
 router
   .get('/', async (req, res) => {
-    const cat = await Cat.getAll();
-    console.log(cat);
-    res.json(cat);
+    const cats = await Cat.getAll();
+    const ids = cats.map((cat) => ({ id: cat.id, name: cat.name }));
```

Comment:
You no longer need the .map().  your getAll is handling the mapping inside the model.